modules/ROOT/pages/clauses/listing-settings.adoc

```diff
@@ -227,6 +227,10 @@ LIMIT 10
 | "20s"
 | "Time out for protocol negotiation handshake."
 
+| "dbms.cluster.network.connect_timeout"
+| "30s"
+| "The maximum amount of time we wait to establish a network connection."
```

Comment:
I've updated the description of this setting. See https://trello.com/c/aWrYCE5Y/213-document-dbmsclusternetworkconnecttimeout

Reply:
And also, I think it wil be introduced in 5.17.